### PR TITLE
auto fetch the latest version of @swc/wasm-web

### PR DESCRIPTION
**Pull request by IWANABETHATGUY**

### CODE REVIEW

File: src/swc.ts [hunk 1]
```diff
@@ -265,6 +282,14 @@ export async function loadSwc(version: string): Promise<SwcModule> {
   return module
 }
 
+export async function getLatestVersion() {
+  const res = await fetch(
+    'https://data.jsdelivr.com/v1/package/npm/@swc/wasm-web'
+  )
+  const data = await res.json()
+  return data.tags.latest ?? ''
```
Comment (g-plane) [inline src/swc.ts]:
```suggestion
  return data.tags.latest
```

File: src/swc.ts [hunk 2]
```diff
@@ -252,10 +252,27 @@ export interface TransformationOutput {
...
+      return versionFromUrl
+    }
+    const latestVersion = await getLatestVersion()
+    if (latestVersion) {
+      return latestVersion
+    } else {
+      throw new Error("can't get the latest version of @swc/wasm-web")
+    }
+  } catch (err) {
+    console.error(err)
```
Comment (g-plane) [inline src/swc.ts]: Personally I think this error can be ignored.

```suggestion
  } catch {
```

File: src/App.tsx [hunk 3]
```diff
@@ -8,7 +9,9 @@ export default function App() {
   return (
     <Box minHeight="100vh" bg={bg}>
       <HeaderBar />
-      <Workspace />
+      <Suspense fallback={<div></div>}>
```
Comment (g-plane) [inline src/App.tsx]: Is it really OK to leave an empty UI for fallback of `Suspense`?

Comment (IWANABETHATGUY) [reply src/App.tsx]: what about a spinner?

Comment (g-plane) [reply src/App.tsx]: There's already something like that: https://github.com/swc-project/swc-playground/blob/main/src/components/Workspace.tsx#L62-L70 . If we can't determine SWC version before finishing fetching versions list, we can just omit version string after "Loading swc" text.

**Comment by g-plane** [comment]
Everyone must sign CLA.

**Pull request closed without merging**